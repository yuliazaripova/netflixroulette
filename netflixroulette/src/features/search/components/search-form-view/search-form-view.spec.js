import * as React from 'react';

import SearchFormView from './search-form-view';

describe("SearchFormView component", () => {
  it("should render SearchFormView component", () => {
    const component = shallow(<SearchFormView />);
    expect(component).toMatchSnapshot();
  })
});
