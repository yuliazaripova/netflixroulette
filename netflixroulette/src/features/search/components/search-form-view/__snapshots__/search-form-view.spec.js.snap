// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchFormView component should render SearchFormView component 1`] = `
<div
  className="search-form_container"
>
  <form
    className="search-form"
  >
    <label
      className="search-form_label"
      htmlFor="search-form_input"
    >
      Find your movie
    </label>
    <div
      className="search-form_input-wrapper"
    >
      <input
        id="search-form_input"
        placeholder="Enter the movie"
        type="text"
      />
    </div>
    <div
      className="search-form_btn-container"
    >
      <RadioToolbar
        items={
          Array [
            Object {
              "id": "title",
              "isChecked": true,
            },
            Object {
              "id": "genres",
              "value": "genre",
            },
          ]
        }
        label="search by"
        name="search-form"
      />
      <input
        className="search-form_submit-btn"
        type="submit"
        value="Search"
      />
    </div>
  </form>
</div>
`;
